lib/broken_record/slack_aggregator.rb

```diff
@@ -8,12 +9,33 @@ def report_final_results
         username: "#{app_name} ValidationMaster"
       })
 
+      send_summary(notifier)
+      send_snippet(notifier)
+    end
+
+    private
+
+    def send_summary(notifier)
       if success?
         notifier.send!("\nAll models validated successfully.")
       else
         notifier.send!("\n#{@total_errors} errors were found while running validations.")
       end
     end
 
+    def send_snippet(notifier)
+      # Our snippet should naively show the output from the ConsoleAggregator
+      console_aggregator = BrokenRecord::ConsoleAggregator.new
+      all_results.each{ |result| console_aggregator.add_result(result) }
+      # Store the output of the console aggregator into a StringIO object
+      $stdout = StringIO.new
```

Comment:
Which part are you not sure of how it works?  It actually isn't overriding -- `send_snippet` is just a wrapper around the notifier `send_snippet!`.  What I'm doing is capturing the output of the console aggregator and sending it via the slack message.